### PR TITLE
fixes spacings in TOC

### PR DESCRIPTION
**Pull request by voisardf**
this PR addresses:
- [x] Issue https://github.com/openoereb/pyramid_oereb_mfp/issues/18
- [x] Issue https://github.com/openoereb/pyramid_oereb_mfp/issues/24

### CODE REVIEW

**Review by vvmruder — COMMENTED** [review]
Maybe you can attach a resulting pdf? :grimacing:

**Comment by voisardf** [comment]
example pdf: 
[20191011015222_extract.pdf](https://github.com/openoereb/pyramid_oereb_mfp/files/3717407/20191011015222_extract.pdf)

**Review by vvmruder — APPROVED** [review]
Looks good, thanks a lot for millimeterizing :laughing:

**Review by kdeininger — APPROVED** [review]

File: print-apps/oereb/toc.jrxml [hunk 1]
```diff
@@ -241,92 +246,103 @@
...
 				<textElement>
 					<font fontName="Cadastra" size="6" isBold="true"/>
 				</textElement>
 				<textFieldExpression><![CDATA[$R{GeneralInformationLabel}]]></textFieldExpression>
 			</textField>
 			<subreport>
-				<reportElement x="260" y="24" width="233" height="21" isPrintWhenDetailOverflows="true" uuid="9fd65c2b-a447-43f1-88f6-9a41a9ca02be">
-					<property name="com.jaspersoft.studio.unit.y" value="pixel"/>
+				<reportElement x="260" y="39" width="233" height="21" isPrintWhenDetailOverflows="true" uuid="9fd65c2b-a447-43f1-88f6-9a41a9ca02be">
+					<property name="com.jaspersoft.studio.unit.y" value="px"/>
```
Comment (marionb) [inline print-apps/oereb/toc.jrxml:325]: **For info:**
The change on this line should not make any difference. Both `px` and `pixel` (as well as `pixels`) are accepted values for the unit pixel by Jaspersoft templates.

**Review by marionb — APPROVED** [review]

File: print-apps/oereb/toc.jrxml (hunk 1 above)
Comment (voisardf) [reply print-apps/oereb/toc.jrxml:325]: thanks. I think it's the way the windows version sets the units? I didn't change it on purpose, Jasper did :)

**Comment by voisardf** [comment]
merging

**Pull request merged**